.buildkite/core.rayci.yml

```diff
@@ -239,6 +239,7 @@ steps:
     if: build.env("BUILDKITE_PIPELINE_ID") == "0189e759-8c96-4302-b6b5-b4274406bf89"
     soft_fail: true
     commands:
-      - bazel run //ci/ray_ci:test_in_docker -- //... core --run-flaky-tests --except-tags manual
+      - bazel run //ci/ray_ci:test_in_docker -- //... core 
+        --run-flaky-tests  --build-type clang
```

Comment:
add `--build-type clang` to run cpp tests; it doesn't affect the python tests in anyway